### ISSUE TITLE
Can I train the output of keras by tensorflow?

### ISSUE THREAD

**Issue by Leo-xh** [keras-team/keras#8586]
If I use keras to get the predictions, only by the layers and the other ops, without using model.
Can I use tf to compute the loss and train the network?
I use them this way but it seemed that the network wasn't trained.

**Comment by harshini-gadige** [comment]
Yes you can use tensorflow to train the network.  Below link would help you to train using tf.train

[Train a Keras model using TF](https://www.tensorflow.org/guide/keras#set_up_training)
